test/browser/component.js

```diff
@@ -316,6 +316,20 @@ describe(`Nested Component instance`, function() {
       childEl.update({animal: `capybara`});
       expect(el.state.animal).to.equal(`capybara`);
     });
+
+    it(`flushes child state updates to parent`, async function() {
```

Comment:
:+1: other test cases to add:
- verify that this same behavior occurs with `appState`
- full "integration" style test with component created via `.child()`

Reply:
Thanks for the feedback! I've added other test cases in 7eccefd